advertising101.md

```diff
@@ -58,6 +59,47 @@ Getting this wrong can be costly, damaging, or potentially even illegal. Adverti
 
+### Why is digital marketing socially important?
+
+Imagine digital marketing disappeared, perhaps because of legislation or technological changes. Who would be hurt? Obviously
+the technology companies who profit off digital advertising would disappear but there would also be other less direct 
+casualties, in particular:
+
+##### Publishers
+
+About a third (35%) of [newspaper ad revenue now comes](https://www.pewresearch.org/fact-tank/2019/07/23/key-takeaways-state-of-the-news-media-2018/) 
+from digital advertising. [Research shows](http://www.digitalnewsreport.org/survey/2019/overview-key-findings-2019/) that 
+only about 11% of people are willing to pay for news subscriptions, and that half of all U.S. news subscriptions go to 
+the Washington Post or New York Times, which means smaller news outlets are disproportionately dependent on ad revenue.
+
+Smaller news outlets have already been struggling over the last decade, and the business models they currently have are 
+highly tenuous. The current COVID-19 pandemic has proven to be a grim natural experiment highlighting just how tenuous 
+those business models are. The economic and media conditions surrounding COVID-19 have lead to [reductions of 1/3 to 1/2 in advertising spending](https://www.nytimes.com/2020/04/14/business/los-angeles-times-furloughs-cuts.html)
+in news media, which has lead to [layoffs and furloughs affecting ~36,000 news media employees](https://www.nytimes.com/2020/04/10/business/media/news-media-coronavirus-jobs.html) (and counting).
+The current loss of revenue is still *less than* the decline of 50-60% in revenue that [Google itself has estimated](https://services.google.com/fh/files/misc/disabling_third-party_cookies_publisher_revenue.pdf) publishers stand to lose
+if current cookie-based advertising techniques are disabled and not adequately replaced.  
+
+A robust free press is essential for a free society, and the explosion of freely available information about every topic
+ imaginable on the web has been an incredible boon to human development and happiness. For as long as newspapers have existed 
+ (all the way back to the 1700's) newspapers
+ have relied on advertising to fund journalism and distribution. 
+ As more and more media moves online, non-digital advertising is waning rapidly. If all advertising moves to digital channels which independent or sub-scale
+ publishers can't access then those publishers (and the numerous social benefits they provide) will simply disappear.
```

Comment:
I may need to make this section (and the one about consumers below) less polemical.